libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/TripNotificationState.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapbox.navigation.base.trip.model
+
+import com.mapbox.api.directions.v5.models.BannerInstructions
+
+data class TripNotificationState(
```

Comment:
How about an internal constructor and factory in an internal package so that we can extend this in the future? Refs https://github.com/mapbox/mapbox-navigation-android/issues/4414.

Reply:
added